modules/compute/event-hub/README.md

```diff
@@ -1,36 +1,35 @@
+| `location`                         | `string` | No       | Required. Location for all resources.                                                                                         |
+| `tags`                             | `object` | No       | Optional. Tags of the resource.                                                                                               |
+| `namespaces`                       | `array`  | No       | Required. The list of the event hub namespaces with its configurations to be created.                                         |
+| `namespaceAuthorizationRules`      | `array`  | No       | Optional. Authorization Rules for the Event Hub Namespace.                                                                    |
+| `namespaceRoleAssignments`         | `array`  | No       | Optional. Role assignments for the namespace.                                                                                 |
+| `namespaceDisasterRecoveryConfigs` | `array`  | No       | Optional. The disaster recovery config for the namespace.                                                                     |
+| `namespaceDiagnosticSettings`      | `array`  | No       | Optional. The Diagnostics Settings config for the namespace.                                                                  |
+| `eventHubs`                        | `array`  | No       | Optional. Name for the eventhub with its all configurations to be created.                                                    |
+| `eventHubAuthorizationRules`       | `array`  | No       | Optional. Authorization Rules for the Event Hub .                                                                             |
+| `eventHubConsumerGroups`           | `array`  | No       | Optional. consumer groups for the Event Hub .                                                                                 |
+| `namespacePrivateEndpoints`        | `array`  | No       | Private Endpoints that should be created for Azure EventHub Namespaces.                                                       |
 
 ## Outputs
 
-| Name                     | Type   | Description                                               |
-| :----------------------- | :----: | :-------------------------------------------------------- |
-| resourceGroupName        | string | The resource group the Azure Event Hub was deployed into. |
-| eventHubNamespaceDetails | array  | Azure Event Hub namespace details.                        |
+| Name                           | Type    | Description                             |
+| :----------------------------- | :-----: | :-------------------------------------- |
+| `eventHubNamespaceNames`       | `array` | Azure Event Hub namespace names.        |
+| `eventHubNamespaceResourceIds` | `array` | Azure Event Hub namespace resource Ids. |
 
 ## Examples
```

Comment:
Can you update the version numbers to `2.0.1` for the examples?

Reply:
Fixed.